src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx

```diff
@@ -1,16 +1,233 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { TypeComponentProps } from "./TypeComponentProps";
-import { InputLabel } from "@mui/material";
+import { InputLabel, MenuItem as MuiMenuItem } from "@mui/material";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import Box from "@mui/material/Box";
-import { DateTimeField, LocalizationProvider } from "@mui/x-date-pickers";
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import timezone from "dayjs/plugin/timezone";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import {
+  Select,
+  DateField,
+  TimeField,
+} from "@madie/madie-design-system/dist/react";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.extend(advancedFormat);
+dayjs.utc().format();
+const userTimeZone = dayjs.tz.guess();
+
+interface MenuObj {
+  value: string;
+  label: string;
+}
+const options: MenuObj[] = [
+  { value: "America/New_York", label: "EST" },
+  { value: "America/Chicago", label: "CST" },
+  { value: "America/Denver", label: "MST" },
+  { value: "America/Los_Angeles", label: "PST" },
+];
+
+const offSets = [
+  { id: "America/New_York - EST", value: "-05:00" },
+  { id: "America/Chicago - CST", value: "-06:00" },
+  { id: "America/Denver - MST", value: "-07:00" },
+  { id: "America/Los_Angeles - PST", value: "-08:00" },
+];
+
```

Comment:
Were these timezones and offsets confirmed with POs? Are we sure we should limit only to these four? (Even if only limited to US, what about Hawaii, Alaska, etc?)

Reply:
@nmorasb Thanks Nate! I posted the question to POs in the jira card.

Reply:
approving for now, but let's keep this jira in code review until we have an answer on that. At least then if any change are needed, the subsequent PR will be more limited